R/utils.R

```diff
@@ -396,6 +396,44 @@ create_query_controls <- function(svy_lkup,
                     "reporting_level", "ppp", "version",
                     "format", "table", "long_format"),
          type = "character")
+
+  # cum_welfare
+  cum_welfare <- list(
+    values = c(min = 0, max = 1),
+    type = "numeric"
+  )
+  # cum_population
+  cum_population <- list(
+    values = c(min = 0, max = 1),
+    type = "numeric"
+  )
+  # requested_mean
+  requested_mean <- list(
+    values = c(min = 0, max = 100),
+    type = "numeric"
+  )
+
+  # mean
+  mean <- list(
+    values = c(min = 0, max = 100),
+    type = "numeric"
+  )
+
+  # times_mean
+  times_mean <- list(
+    values = c(min = 0, max = 1),

```

Comment:
The min can't be zero. Also, I don't think this is parameter that should be included in the endpoint. This is an internal parameter in `pipster`. Does not have much functionality here. However, if it is already incorporated, do not take it away. Just make sure the min is `0.01` (1%), and the max value times the mean Can't be higher than the max value of the allowed poverty line, so this is trick. I would add a max of `5` and that's it.